tce/src/codesign/Proxim/ProximMachineStateWindow.cc

```diff
@@ -512,11 +512,11 @@ ProximMachineStateWindow::onExport(wxCommandEvent&) {
     wxString defaultDir = _T(".");
     wxString defaultFile= _T("");
 #if wxCHECK_VERSION(3, 0, 0)
-    wxString fileTypes = _T("Portable Network Graphics (.png)|*.png");
```

Comment:
Cannot .png export be supported as well? Would be nice to have a bitmap and a vector format supported? If it's too much trouble, not important.